secretmanager/apiv1/iam.go

```diff
@@ -1,29 +0,0 @@
-// Copyright 2020 Google LLC
```

Comment:
@adrianajg We will need to add these two iam files to our keep list in owlbot configuration

Reply:
Done

Reply:
Got it